mmcv/cnn/bricks/transformer.py

```diff
@@ -568,6 +600,8 @@ class FFN(BaseModule):
             zeroed in FFN. Default 0.0.
         add_identity (bool, optional): Whether to add the
             identity connection. Default: `True`.
+        use_layer_scale (bool): Whether to use layer_scale in FFN.
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.

Reply:
BTW, is it proper to add a parameter for FFN? Should we define a new class named `DeiT3FFN` which subclasses `FFN`.

Reply:
Since layer_scale is often used in modern transformer models, it would be nice if FFN had a param for layer_scale.